apps/home/models.py

```diff
@@ -78,7 +78,8 @@ class HomePage(Page):
         ObjectList(de_ls_content_panels, heading='Easy German'),
     ])
 
-    subpage_types = ['apps_home.DetailPage',
+    subpage_types = ['apps_home.OverviewPage',
+                     'apps_home.DetailPage',
```

Comment:
We could also (or only) add the detail pages as subpages to the overview page. Maybe it's easier to keep an overview about the pages then?!

Reply:
Yes, I was also wondering about that, but then wasnt sure if we would maybe still need the detail page as subpage of homepage somewhere? But adding them as subpages to overview page definitely makes sense!